pom.xml

```diff
@@ -612,7 +612,6 @@
         <groupId>xml-apis</groupId>
         <artifactId>xml-apis</artifactId>
         <version>1.4.01</version>
-        <scope>test</scope>
```

Comment:
@srowen, I was wondering this actually overrides the scope of `xml-apis`, forces to be present in `test` scope only, and make it not present in the build output.